package.json

```diff
@@ -127,5 +127,6 @@
     "lines": 100,
     "functions": 100,
     "statements": 100
-  }
+  },
+  "sideEffects": false
```

Comment:
No harm to have this in the root package.json.

Reply:
Hmm. `dist/diff.js` and `dist/diff.min.js` *do* have side effects - they create the `Diff` global. *Probably* this doesn't matter; I think there's no way anybody could be consuming those files that would respect `sideEffects: false`, given that they're not even exported via our `exports` property in `package.json`. But I am not *100%* sure of that. If it's not serving a purpose, I am inclined not to add this line here, just in case. (Leaving out the one that doesn't do anything also makes it clear which property is actually having an effect, if we want to change it later.)